amplitude-api.gemspec

```diff
@@ -21,6 +21,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rspec', '~> 2.99', '>= 2.99.0'
   spec.add_development_dependency 'rake', '~> 10.0', '>= 10.0'
   spec.add_development_dependency 'pry', '~> 0.9.12.6'
-  spec.add_dependency 'typhoeus', '~> 1.0.2'
+  spec.add_dependency 'typhoeus', '~> 1.1.2'
```

Comment:
@rushilagr , maybe `~> 1.0` ?

Reply:
1.0.0 for typhoeus? that would be a downgrade. We should go with the latest. It works smoothly with the gem

Reply:
@rushilagr , see:

http://bundler.io/v1.5/gemfile.html#gemfiles

> The specifier ~> has a special meaning, best shown by example. ~> 2.0.3 is identical to >= 2.0.3 and < 2.1. ~> 2.1 is identical to >= 2.1 and < 3.0.

so, `~> 1.0`  is identical to  >= 1.0 and < 2.0.

Also, https://github.com/typhoeus/typhoeus/releases/tag/v1.3.0
can you try with last version? Will it work?

@toothrot , what do you think?

Or better to use `>= 1.0.2`?

Reply:
Oh yes, my bad. ~> 1.0 should do just fine.
Will have a look with 1.3.0